core/jdiameter/api/src/main/java/org/jdiameter/api/sy/events/SessionTerminationRequest.java

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
Why not reuse https://github.com/RestComm/jdiameter/blob/master/core/jdiameter/api/src/main/java/org/jdiameter/api/auth/events/SessionTermRequest.java ?